Packs/Indeni/Integrations/Indeni/Indeni.yml

```diff
@@ -0,0 +1,207 @@
+category: Network Security
+commonfields:
+  id: Indeni
+  version: -1
+configuration:
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- additionalinfo: If true, only Palo Alto Networks Vulnerability issues will be pulled.
+    If false, all issues will be pulled.
+  defaultvalue: 'true'
+  display: Only Pull Palo Alto Networks Vulnerability Issues
+  name: onlyPullPanCveIssues
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: CRITICAL
```

Comment:
I dont think I make much sense, because if a customer wants to set it up and test, It may probably filter the alerts and it will be confusing to understand why nothing is fetched.
Think it is better to use  'WARN' or 'INFO' by default.

Reply:
Done

Reply:
@yuvalbenshalom  can you please elaborate the comment about the .yml file?

Reply:
@fvigo It was connected to the comment that I asked to add from version: 5.0.0
for some reason it is now comment without a context. never mind I will add it.